Makefile

```diff
@@ -364,8 +364,8 @@ ifeq ($(LINUXKIT_PKG_TARGET),push)
 endif
 
 # Though the partition size is set to 512MB lets check for ROOTFS_MAXSIZE_MB not exceeding 450MB for kubevirt.
-# That seems to be the direction taken for existing kvm systems where partition size is 300MB but
-# rootfs size is limited to 250MB. That helps in catching image size increases earlier than at later stage.
+# That seems to be the direction taken for existing kvm systems where partition size is 512MB but
```

Comment:
512 are the new (since release TBD) but all the older use 300. The comments need to reflect this.